docs/customization/excluding-packages-and-paths.md

```diff
@@ -0,0 +1,51 @@
+# Excluding packages or sources/models based on their path
+
+!!! note
+
+    This section is describing how to entirely exclude models/sources and packages to be evaluated.
+    If you want to document exception to the rules, see the section [on exceptions](exceptions.md)
+    and if you want to deactivate entire tests you can follow instructions from [this page](customization.md)
+
+There might be cases where you might want to exclude models/sources from being tested:
+
+- they could come from a package for which you have no control over
+- you might be refactoring your project and wanting to exclude entire folders to follow best-practices in the new models
+
+In that case, this package provides the ability to exclude whole packages and/or models and sources based on their path
+
+## Configuration
+
+The variables `exclude_packages` and `exclude_paths_from_project` allow you to define a list of regex patterns to exclude from being reported as errors.
+
+- `exclude_packages` accepts a list of package names to exclude from the tool. To exclude all packages except the current project, you can set it to `["all"]`
+- `exclude_paths_from_project` accepts a list of regular expressions of paths to exclude for the current project
+    - **for models**, the regex provided will try to match the pattern in the string `<path/to/model.sql>`, allowing to exclude packages, but also whole folders or individual models
+    - **for sources**, the regex will try to match the pattern in `<path/to/sources.yml>:<source_name>.<source_table_name>` *(the pattern is different than for models because the path itself doesn't let us exclude individual sources)*
+
+### Example to exclude a whole package
+
+```yaml title="dbt_project.yml"
+vars:
+  exclude_packages: ["<package_name>"]
```

Comment:
I would prefer if we had hard-coded examples for people to reference here, instead of <package_name>. 

Same for the below examples with paths.